README.html

```diff
@@ -20,117 +20,199 @@
-                </li>
-            </ul>
-
-            <h3 style="text-align: center;">For developers</h3>
-            <ul>
-                <li>
-                    Refer to CONTRIBUTING.md and DEVELOPMENT.md for more information on how to contribute to this
-                    project.
-                </li>
-            </ul>
-            <hr>
-
-            <h2 style="text-align: center;">Disclaimer</h2>
-            This application is vulnerable! There are documented vulnerabilities that will cause more undocumented
-            ones.<br>
-            So, it is strictly advised <b>not</b> to use it directly on your host machine without any virtual machine or
-            docker.<br>
-            Do not host code from this project on internet-facing servers.<br><br>
+            provided to learn about:</p>
+        <ul>
+            <li>The vulnerability</li>
+            <li>How it can be exploited</li>
+            <li>The part of the code causing this vulnerability.</li>
+            <li>How to patch this vulnerability at different levels.</li>
+        </ul>
+
+        <h2>Installation</h2>
+        <ol>
+            <li>
+                To set up this application on Windows, you must run Apache and MySQL services, which are inbuilt in
+                XAMPP.
+            </li>
+            <li>
+                Follow this for setting up Cyber Sentinel on Windows.
+            </li>
+            <li>
+                After setting up and downloading the repository, rename the folder to cybersentinel and place it inside
+                htdocs
+            </li>
+            <li>
+                You can also download the dockerized version of the application here, which will require you to download
+                the Docker Desktop application for Windows.
+            </li>
+            <li>
+                Follow the steps in this Docker Hub repository to use this application on Docker.
+            </li>
+
+        </ol>
+
+
+
+
+
+        <h2>For users</h2>
+        <ul>
+            <li>
+                There are different vulnerabilities with different difficulty levels. The difficulty levels are based on
+                how good the patch is for that specific vulnerability.
+            </li>
+            <li>
+                There is no fixed objective to complete a module. If you feel you've exploited the system thoroughly,
+                the goal is reached!
+            </li>
+            <li>
+                There is a help button at the bottom to view hints & tips for that vulnerability. There are additional
+                links for further reading on each vulnerability.
+            </li>
+        </ul>
+
+
+        <h2>For developers</h2>
+        <ul>
+            <li>
+                Refer to CONTRIBUTING.md and DEVELOPMENT.md for more information on how to contribute to this project.
+            </li>
+        </ul>
+
+        <h2 style="text-align: center;">Disclaimer</h2>
```

Comment:
From here onwards, text-align center is left unremoved.

Reply:
i did this intentionally  , if you didnt liked the idea we can change it back

Reply:
It does look better this way. Let's leave it like this.